components/Footer.tsx

```diff
@@ -7,9 +7,9 @@ export default function Footer() {
   const lang = useLang()
   const t = useTranslations(lang)
   return (
-    <div className="h-auto w-full">
-      <div className="flex flex-col items-center justify-center px-6 py-4 text-white sm:flex-row">
-        <p className="p-1">{t('footer.paragraph_one')}</p>
+    <div className="bottom-0 h-auto w-full bg-back">
```

Comment:
I don't think you need `bottom-0`, `h-auto`, and `w-full` here. `bottom-0` has no effect since the element position is relative. `h-auto` and `w-full` are determined by the flex layout of the parent container.

Unnecessary layout classes is something I generally see a good amount in this codebase overall. Not a big issue if things look good, but also results a bit in spaghetti code. Not sure if I should point these out in general or only if they cause issues... what do you think?

Reply:
Feel free to point these out @GBKS, you have keener eye for these than I do. Fewer css classes ultimately means fewer bytes we have to send over the network to our clients, resulting in better pagespeed.